### ISSUE TITLE
LCM Triomics Pipeline configuration and integration test for EMSeq branch

### ISSUE THREAD

**Issue by yoldas** [sanger/sequencescape#4288]
**User story**
As PSD, we would like to create configuration and integration test for the EMSeq branch of the LCM Triomics WGS/EMSeq pipeline to enable parallel development of the EMSeq branch of the pipeline, lower part of the Process Diagram.

**Who are the primary contacts for this story**
Abdullah (PSD)
Katy (PSD)
Shiv (PSD)
Scott G (R&D)
Sara W (Lab ops)

**Who is the nominated tester for UAT**
e.g. John S (don't include surnames in public repos)

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Sequencescape and Limber configuration of the EMSeq branch
- [ ] Integration Suite test with placeholder manual transfers where the bed verifications will go (separate story)

**Dependencies**
This story is blocked by the following dependencies:
None

References
This story is part of the parent story:
- [Y24-194 - LCM WGS/EM-Seq - Pipeline configuration and integration test]

**Additional context**
We do a common initial submission for WGS and EMSeq branches.

We allow pre-creation of the following plates using Permissive presenter: LCMT DNA frag, LCMT DNA end prep, LCMT DNA adp lig, LCMT EM ox and LCMT EM ox stop

**Comment by yoldas** [comment]
Current WIP names

WIP=013_lcm_triomics_plate_purposes,013_lcm_triomics_catalogue,018_limber_lcm_triomics_request_types,014_lcm_triomics_submission_templates

**Comment by yoldas** [comment]
GitLab Integration suite merge request: [Y24 266 lcm triomics pipeline configuration and integration test for emseq branch](https://gitlab.internal.sanger.ac.uk/psd/integration-suite/-/merge_requests/158)

The branch name is `y24-266---lcm-triomics-pipeline-configuration-and-integration-test-for-emseq-branch`

**Comment by yoldas** [comment]
Tag Layout Template Names are common for both branches; received by email. I have added them to the `LCMT EM LibPCR` purpose config.

	TS_pWGSA_UDI96v2 (also known as pWGSA_UDI_tempi7s and pWGSA_UDI_tempi5s)
	TS_pWGSB_UDI96
	TS_pWGSC_UDI96
	TS_pWGSC_UDI_tag60_61_swap
	TS_pWGSD_UDI96
	
 pWGSA_UDI_tempi7s and pWGSA_UDI_tempi5s refer to the tag groups for the first one. See the query in the screenshot.
 
![image](https://github.com/user-attachments/assets/467ea7ac-2f76-4135-8123-6d7806b7d0de)

**Comment by yoldas** [comment]
For UAT Testing:

We have deployed the EMSeq branch of the LCM Triomics Pipeline ([Y24-271](https://github.com/sanger/sequencescape/issues/4291) and [Y24-266](https://github.com/sanger/sequencescape/issues/4288)) into the UAT environment. 

There is no bed verification in this story, which will be implemented by [Y24-198](https://github.com/sanger/limber/issues/1782).

EMSeq branch is the lower path in the process map. Note that there will be another branch off Adp Lig plate but it is a separate story ([Y24-267](https://github.com/sanger/sequencescape/issues/4289))

- The EMSeq branch starts with the submission `Limber-Htp - LCM Triomics EMSeq` on `LCMT Lysate` plate.
- The request type for this submission is `LCM Triomics EMSeq.` Note that this is a library request which is used until the end of the EMSeq branch. We could not have a separate DNA Prep request for the common part with WGS because of pre-creating plates across the branch point.
- The library type is set to `LCM Triomics EMSeq` for the branch.
- The EMSeq branch allows creation of the following plates without any transfers: `LCMT DNA Frag, LCMT DNA End Prep, LCMT DNA Adp Lig, LCMT EM TET2 Ox, and LCMT EM TET2 Stop` .
- Once they are created, manual transfers into them can happen in that order (no bed verification in this story)
- Concentrations for LCMT EM PCR XP plate can be added by "Generate concentrations for a plate" UAT action for the test. (Quanthub upload of concentrations)
- A tag plate is coming into the LCMT EM Lib PCR plate.
- The LCMT EM Pool XP tube needs a custom pooling file.
- The pipeline continues until the end to create the `LCMT EM Pool XP Norm `tube.

UAT environment:
Sequencescape: https://uat.sequencescape.psd.sanger.ac.uk/
Limber: https://uat.limber.psd.sanger.ac.uk/
